### DISCUSSION TITLE
bing对于较长的回复，无法做出完整的回应，有无解决办法？

### DISCUSSION THREAD

**Discussion by Yimo-Senior** [viopsa233/AI-testers-Forum#14] [Q&A]
有时候，我们在提问bing的时候，它的回复到达一定的长度就会戛然而止；有时候你让他继续回复，回复结果会不尽人意，回复的内容和上一次回复虽然有些联系但是会偏离主题。

**Comment by Yimo-Senior** [comment]
我在想，是否有办法让他将回复进行到底。

**Comment by bing2023x** [comment]
az不会吧，直接无脑发继续救行

**Comment by TiansuoLi** [comment]
如果正在做总结或者讲故事，直接说“请继续”就行了呀。不过总文字量超过一定量之后AI确实会逻辑混乱越来越难继续，这个正常
